### ISSUE TITLE
NTR: Mycobacteria PCR in synovial biopsy positive (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8085]
New term request Mycobacteria PCR in synovial biopsy positive (athrey_140304135600): Polymerase chain reaction; DNA

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000735
- label: Positive mycobacterial synovial nucleic acid test
- parents: [Positive synonvial fluid infectious agent test (HP:6000581)](https://hpo.jax.org/app/browse/term/HP:6000581)
- pmid: 
- def: Detection of nucleic acid of mycobacteria in the synovial fluid by a method such as polymerase chain reaction (PCR).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8085
- synonyms:
